lib/ansible/plugins/connection/ssh.py

```diff
@@ -805,26 +805,37 @@ def _file_transport_command(self, in_path, out_path, sftp_action):
                 cmd = self._build_command('sftp', to_bytes(host))
                 in_data = u"{0} {1} {2}\n".format(sftp_action, shlex_quote(in_path), shlex_quote(out_path))
                 in_data = to_bytes(in_data, nonstring='passthru')
-                (returncode, stdout, stderr) = self._run(cmd, in_data, checkrc=False)
+                try:
+                    (returncode, stdout, stderr) = self._run(cmd, in_data, checkrc=False)
+                except AnsibleConnectionFailure:
+                    returncode = 255
             elif method == 'scp':
                 if sftp_action == 'get':
                     cmd = self._build_command('scp', u'{0}:{1}'.format(host, shlex_quote(in_path)), out_path)
                 else:
                     cmd = self._build_command('scp', in_path, u'{0}:{1}'.format(host, shlex_quote(out_path)))
                 in_data = None
-                (returncode, stdout, stderr) = self._run(cmd, in_data, checkrc=False)
+                try:
+                    (returncode, stdout, stderr) = self._run(cmd, in_data, checkrc=False)
+                except AnsibleConnectionFailure:
```

Comment:
You seem to be missing a `returncode = 255` here.  As such, this PR contains a SyntaxError.